CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- If the package root contains a file `index.rst`, it is used instead of the README to
```

Comment:
index.rst is not a super nice name... as it usually refers to the "main page" of the documentation.

Hence I would check for some doc/doc_readme.rst? or doc_readme.rst?

WDYT?

Reply:
Well, it _is_ supposed to be the documentation main page (breathing-cat basically takes this file and just appends the stuff for API docs, etc.) and explicitly not a README (which IMO serves a different purpose).

We could think about different names, if you think `index.rst` is confusing, maybe something like `doc_mainpage.rst`?  Unfortunately it needs to be in the package root, not in `doc/` as otherwise it gets very confusing with relative links.

Reply:
ok for doc_mainpage.rst

for the doc/ folder I am not sure about it but let's keep it in the root folder for now

Reply:
Done.
I also added support for `doc_mainpage.md` (as I realised, this is just one line to add) and refactored a bit.